### PR TITLE
3.7 release blog post

### PR DESCRIPTION
**Pull request by christinaforney**

### CODE REVIEW

File: blogposts/announcing-sourcegraph-3.7.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+You can now set `EXTSVC_CONFIG_ALLOW_EDITS=true` and `SITE_CONFIG_ALLOW_EDITS=true` to allow the active Sourcegraph configuration to diverge from the file, thus allowing temporary edits. For complete details, see [the documentation](https://docs.sourcegraph.com/admin/config/advanced_config_file).
+
+## In case you missed it...
+
+- We learned a ton at GopherCon, and you can read about some of the awesome talks from the GopherCon live-blog!
+- We had a fruitful hackweek, with one project resulting in [improved default search on sourcegraph.com]
+- We have been exploring LSIF as a fast and precise code intelligence solution. Read about [what we’ve been learning]!
+
+## 3.7.0 Changelog
```
Comment (christinaforney) [inline blogposts/announcing-sourcegraph-3.7.md]: I was given feedback to include all patch release changes that are bubbling into the release - this ended up looking funny with the headers. Would it be better to add a style for h4, and nest the versions one level deeper under ## Changelog?

Comment (christinaforney) [reply blogposts/announcing-sourcegraph-3.7.md]: I updated the css.

File: blogposts/announcing-sourcegraph-3.7.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+## Sourcegraph configuration temporary overrides in the UI
+
+Sourcegraph 3.4 [introduced an optional way to load Sourcegraph configuration from a file or Kubernetes config map](https://about.sourcegraph.com/blog/sourcegraph-3.4#optional-loading-of-configuration-from-the-file-system-or-kubernetes-configmap). It is helpful for teams with Sourcegraph as a critical piece of infrastructure, to be able to check the Sourcegraph configuration into version control. We received feedback that it would be nice to be able to make temporary edits in the web UI and diverge from the files on disk intentionally (i.e. to try an option out before committing it).
+
+You can now set `EXTSVC_CONFIG_ALLOW_EDITS=true` and `SITE_CONFIG_ALLOW_EDITS=true` to allow the active Sourcegraph configuration to diverge from the file, thus allowing temporary edits. For complete details, see [the documentation](https://docs.sourcegraph.com/admin/config/advanced_config_file).
+
+## In case you missed it...
+
+- We learned a ton at GopherCon, and you can read about some of the awesome talks from the GopherCon live-blog!
```
Comment (christinaforney) [inline blogposts/announcing-sourcegraph-3.7.md]: @ryan-blunden to update copy and add links to blogposts.

File: blogposts/announcing-sourcegraph-3.7.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+- We had a fruitful hackweek, with one project resulting in [improved default search on sourcegraph.com]
+- We have been exploring LSIF as a fast and precise code intelligence solution. Read about [what we’ve been learning]!
```
Comment (sqs) [inline blogposts/announcing-sourcegraph-3.7.md]: Missing link?

**Pull request merged**